core/server/config/index.js

```diff
@@ -94,7 +94,7 @@ ConfigManager.prototype.loadThemes = function () {
 
     return readThemes(self._config.paths.themePath)
         .then(function (result) {
-            self.set({paths: {availableThemes: result}});
+            self._config.paths.availableThemes = result;
```

Comment:
It's funny because i saw this line was changed in one of your PR's, but i thought "yeah that makes sense, nconf uses the set function.........."

Totally confused between LTS and master sometimes 💃